### DISCUSSION TITLE
A tiny mistake on 'Prisma schema - Relations' documentation?

### DISCUSSION THREAD

**Discussion by slow-loop** [prisma/prisma#2147] [Q&A] [answered]
Just reading the [documentation](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/relations), and found this part may have a mistake:
```
model User {
  id        Int      @id @default(autoincrement())
  posts     Post[]
}
model Post {
  id        Int    @id @default(autoincrement())
  author    User?  @relation(fields: [authorId], references: [author])
                 >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>references: [id] ???
  authorId  Int?
}
```

**Comment by ryands17** [comment, marked as answer]
Hey @w121211 you're right! I have made the necessary updates.
Thank you for noticing this :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
